system/Database/BaseBuilder.php

```diff
@@ -2698,7 +2698,7 @@ protected function objectToArray($object)
 
         foreach (get_object_vars($object) as $key => $val) {
             // There are some built in keys we need to ignore for this conversion
-            if (! is_object($val) && ! is_array($val) && $key !== '_parent_name') {
+            if (((! is_object($val) || $val instanceof RawSql) && ! is_array($val) && $key !== '_parent_name')) {
```

Comment:
It seems to me that there are too many parentheses.

Reply:
Your right, thanks